### PR TITLE
Add cmd script to install with cabal

### PR DESCRIPTION
**Pull request by jneira**
Sorry for restore an os specific script but i've got tired of typing or copypasting it

### CODE REVIEW

**Comment by fendor** [comment]
Well, I am fine with a single line script.

**Comment by alanz** [comment]
I am happy with this too. Should we just put it there, or should there be something about it in the README?

When you invoke it, do you type `cabal-install`, or `cabal-install.cmd`? I am worried about the name, with respect to the package `cabal-install` too.

And lastly,

**Comment by jneira** [comment]
A comment in the readme could be useful, i'll do
You can invoke it without ext: `cabal-install hie-8.6.5` f.e.

**Comment by fendor** [comment]
That is a bit ambiguous, imo. Maybe a name like `cabal-hie-install` or something, making clear this is a hie specific installation script?

File: cabal-hie-install [hunk 1]
```diff
@@ -0,0 +1 @@
+cabal v2-run ./install.hs --project-file install/shake.project $@
```
Comment (alanz) [inline cabal-hie-install]: This should have `#!/bin/sh` on top, surely? and must be `chmod +x`'d too, if not already done.

Comment (jneira) [reply cabal-hie-install]: I already set the executable bit. I could execute as is in my msys bash but i'll add the sehbang just in case.

**Comment by power-fungus** [comment]
I guess we should poke the cabal team to allow setting the cabal project file inside the script. Hvr has hinted that this should be no problem to support but nobody has done that yet

**Comment by jneira** [comment]
> I guess we should poke the cabal team to allow setting the cabal project file inside the script. Hvr has hinted that this should be no problem to support but nobody has done that yet

[The issue i opened](https://github.com/haskell/cabal/issues/6353) has not much feedback 🤔

**Pull request merged**